orca-peering/src/main/kotlin/com/netflix/spinnaker/orca/peering/PeeringAgent.kt

```diff
@@ -160,6 +160,12 @@ class PeeringAgent(
 
     if (pipelineIdsToMigrate.isNotEmpty() || pipelineIdsToDelete.isNotEmpty()) {
       log.debug("Found ${completedPipelineKeys.size} completed $executionType candidates with ${migratedPipelineKeys.size} already copied for peering, ${pipelineIdsToMigrate.size} still need copying and ${pipelineIdsToDelete.size} need to be deleted")
+      val maxDeleteCount = dynamicConfigService.getConfig(java.lang.Integer::class.java, "pollers.peering.max-allowed-delete-count", Integer(1000))
+      if (pipelineIdsToDelete.size > maxDeleteCount.toInt()) {
+        log.error("Number of pipelines to delete (${pipelineIdsToDelete.size}) > threshold ($maxDeleteCount) - not performing deletes - if this is expected you can set the pollers.peering.max-allowed-delete-count property to a larger number")
```

Comment:
for visibility: the third change is a safety mechanism to avoid deleting the world here

Reply:
thank you - amended the commit but github doesn't pick it up, fixing